improver_tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -104,6 +105,23 @@ def setUp(self):
             frt=datetime(2015, 11, 18, 22),
         )
 
+        cube_list = iris.cube.CubeList([])
+        for threshold in [1.0, 2.0]:
+            cube = iris.util.squeeze(self.cube3.copy())
+            cube.coord("lwe_thickness_of_precipitation_amount").points = np.array(
+                [threshold]
+            )
+            cube_list.append(cube)
+        self.cube4 = cube_list.merge_cube()
+        cube_list = iris.cube.CubeList([])
+        for point in np.arange(3):
+            cube = self.cube4.copy()
+            cube.add_aux_coord(
+                iris.coords.DimCoord(np.array([point]), standard_name="realization")
+            )
+            cube_list.append(cube.copy())
+        self.cube4 = cube_list.merge_cube()
```

Comment:
I think you can use the `add_coordinate` function to do this more cleanly.

Reply:
Done